index.js

```diff
@@ -1,268 +0,0 @@
-  if (parseInt(scoreTotalUpper.innerHTML) >= 63) {
-    bonusField.innerHTML = 35;
-    scoreTotal.innerHTML = parseInt(scoreTotal.innerHTML) + 35;
-    scoreTotalUpper.innerHTML = parseInt(scoreTotalUpper.innerHTML) + 35;
-  } else bonusField.innerHTML = 0;
-}
-
-function createDiceImage(num) {
-  const diceImg = document.createElement("img");
-  diceImg.src = `Images/Alea_${num}.png`;
-  return diceImg;
-}
-
-function keepDice() {
-  keepDiceInput.classList.toggle("hidden");
-  for (let i = 0; i < checkboxDice.length; i++) {
-    if (checkboxDice[i].checked) {
-      dice[i].keep = true;
-    } else dice[i].keep = false;
-  }
-  if (rollCount < 3) {
-    rollButton.classList.toggle("hidden");
-  }
-}
-
-function increaseRollCount() {
-  rollCount++;
-  rollCounter.innerHTML = `Roll ${rollCount} / 3`;
-}
-
-function chooseField() {
-  rollCounter.classList.toggle("hidden");
-  scoreInput.classList.toggle("hidden");
-  for (radioButton of radioButtonsScore) {
-    if (radioButton.checked) {
-      selectedField = radioButton.value;
-      radioButton.disabled = true;
-    }
-  }
-  score(selectedField);
-  for (let i = 0; i < allRadioButtons.length; i++) {
-    if (!allRadioButtons[i].disabled) {
-      allRadioButtons[i].checked = true;
-      break;
-    }
-  }
-}
-
-function score(str) {
-  let scoreField = document.getElementById(`score-${str}`);
-  scoreField.innerHTML = evaluate(selectedField);
-  scoreTotal.innerHTML =
-    parseInt(scoreTotal.innerHTML) + evaluate(selectedField);
-  if (
-    str === "ones" ||
-    str === "twos" ||
-    str === "threes" ||
-    str === "fours" ||
-    str === "fives" ||
-    str === "sixes"
-  ) {
-    scoreTotalUpper.innerHTML =
-      parseInt(scoreTotalUpper.innerHTML) + evaluate(selectedField);
-  }
-  dice = [];
-  newRound();
-}
-
-function newRound() {
-  turnCount++;
-  if (turnCount < 14) {
-    rollCounter.classList.toggle("hidden");
-    rollButton.classList.toggle("hidden");
-    clearDiceArray();
-    rollCount = 1;
-    rollCounter.innerHTML = `Roll ${rollCount} / 3`;
-  }
-  if (turnCount >= 14) endOfGame();
-}
-
-function changeDiceImage(elem, num) {
-  const diceImg = createDiceImage(num);
-  elem.appendChild(diceImg);
-}
-
-function changeDiceImageRandom() {
-  dice = dice.map((singleDice) => {
-    if (singleDice.keep) return singleDice;
-    let randomNum = Math.floor(Math.random() * 6 + 1);
-    if (singleDice.diceElement.hasChildNodes()) {
-      singleDice.diceElement.innerHTML = "";
-    }
-    changeDiceImage(singleDice.diceElement, randomNum);
-    return { ...singleDice, value: randomNum };
-  });
-}
-
-function evaluate(str) {
-  const diceValues = dice.map((die) => die.value);
-  const diceValuesSum = diceValues.reduce((a, b) => {
-    return a + b;
-  });
-  const diceValuesSorted = diceValues.reduce((a, b) => {
-    a[b] = (a[b] || 0) + 1;
-    return a;
-  }, {});
-  const diceQuantity = Object.values(diceValuesSorted);
```

Comment:
I would like the plural form here more. but this is subjective. 

it's a very clever way to solve this problem, at first glance it looks like you are trying to determine whether you had thrown a 3, 4 or a 5 for the "three-of-a-kind" for example... 

maybe you can rename these .include checks so that it's not as confusing on a first read... or wrap the includes check in a function that is clearly named. 

another thing I don't like about this evaluate function is that it just assigns the sum or the 0 but you should really have 2 functions -> one to determine the possible options and then a second one that calculates the score from an option +  diceValues ... this way you could limit the options people can select when playing your game. i.e. after each throw, you can calculate possible options - and only show those to the users / highlight them in the scoresheet with css. 

this could work even better when you take into account what is already taken/not taken from the scoresheet :)